### PR TITLE
Feature unit tests

### PR DESCRIPTION
**Pull request by sujata-m**
- increases test coverage to 81%
- Added unit test cases for 
     - Topic
     - FileEntity
     - Queue
     - Storage Classes
     - Logger Classes
     - Configurations
     - Authorizer
NOTE: The enumeration document is not yet complete. Here are the test results from the above code:
```shell
>  coverage run --source=src/python_ms_core -m unittest discover -v tests/unit_tests
test_has_permission (test_auth.abstract.test_authorizer_abstraction.TestAuthorizerAbstract) ... ok
test_get_search_params (test_auth.models.test_permission_request.TestPermissionRequest) ... ok
test_has_permission_with_invalid_permissions (test_auth.provider.test_hosted_authorizer.TestHostedAuthorizer) ... ok
test_has_permission_with_missing_auth_url (test_auth.provider.test_hosted_authorizer.TestHostedAuthorizer) ... ok
test_has_permission_with_valid_permissions (test_auth.provider.test_hosted_authorizer.TestHostedAuthorizer) ... ok
test_has_permission_should_return_should_satisfy_all (test_auth.provider.test_simulated_authorizer.TestSimulatedAuthorizer) ... ok
test_has_permission_should_return_should_satisfy_none (test_auth.provider.test_simulated_authorizer.TestSimulatedAuthorizer) ... ok
test_core_config_auth (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_auth_url (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_auth_url_default (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_logger (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_provider (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_provider_default (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_provider_override (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_queue (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_queue_connection (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_queue_connection_default (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_queue_name (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_queue_name_default (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_storage (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_storage_connection (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_storage_connection_default (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_topic (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_topic_connection (test_config.test_cofig.TestCoreConfig) ... ok
test_core_config_topic_connection_default (test_config.test_cofig.TestCoreConfig) ... ok
test_local_config_auth (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_logger (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_provider (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_provider_default (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_queue (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_queue_connection (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_queue_connection_default (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_queue_name (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_queue_name_default (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_storage (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_storage_connection (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_storage_connection_default (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_topic (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_topic_connection (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_topic_connection_default (test_config.test_cofig.TestLocalConfig) ... ok
test_local_config_auth (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_logger (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_provider (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_queue (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_queue_connection_default (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_queue_name_default (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_storage (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_storage_connection_default (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_topic (test_config.test_cofig.TestUnknownConfig) ... ok
test_unknown_config_topic_connection_default (test_config.test_cofig.TestUnknownConfig) ... ok
test_get_authorizer_no_config (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: SIMULATED
ok
test_get_authorizer_with_config_hosted_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: AZURE
ok
test_get_authorizer_with_config_simulated_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: SIMULATED
ok
test_get_authorizer_with_config_unknown_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: UNKNOWN
ok
test_get_logger_azure_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: AZURE
ok
test_get_logger_local_provider (test_core.TestCore) ... ok
test_get_logger_unknown_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: UNKNOWN
ok
test_get_storage_client_azure_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: AZURE
ok
test_get_storage_client_local_provider (test_core.TestCore) ... ok
test_get_storage_client_unknown_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: UNKNOWN
ok
test_get_topic_azure_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: AZURE
ok
test_get_topic_local_provider (test_core.TestCore) ... ok
test_get_topic_unknown_provider (test_core.TestCore) ... 2023-07-03 18:07:41 ERROR    Failed to initialization core.get_logger for provider: UNKNOWN
ok
test_add_request (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_debug (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_error (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_info (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_record_metric (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_warn (test_logger.abstract.test_logger_abstract.TestLoggerAbstract) ... ok
test_add_request (test_logger.test_local_logger.TestLocalLogger) ... ok
test_debug (test_logger.test_local_logger.TestLocalLogger) ... ok
test_error (test_logger.test_local_logger.TestLocalLogger) ... ok
test_info (test_logger.test_local_logger.TestLocalLogger) ... ok
test_record_metric (test_logger.test_local_logger.TestLocalLogger) ... ok
test_warn (test_logger.test_local_logger.TestLocalLogger) ... ok
test_add_request (test_logger.test_logger.TestLogger) ... ok
test_debug (test_logger.test_logger.TestLogger) ... ok
test_error (test_logger.test_logger.TestLogger) ... ok
test_info (test_logger.test_logger.TestLogger) ... ok
test_record_metric (test_logger.test_logger.TestLogger) ... ok
test_warn (test_logger.test_logger.TestLogger) ... ok
test_concrete_queue_has_abstract_methods_implemented (test_queue.abstract.test_queue_abstract.TestQueueAbstract) ... ok
test_concrete_queue_inherits_from_queue_abstract (test_queue.abstract.test_queue_abstract.TestQueueAbstract) ... ok
test_azure_provider_invalid_connection_string (test_queue.config.test_queue_config.TestConfig) ... ok
test_azure_provider_missing_queue_name (test_queue.config.test_queue_config.TestConfig) ... ok
test_azure_provider_valid_connection_string (test_queue.config.test_queue_config.TestConfig) ... ok
test_non_azure_provider (test_queue.config.test_queue_config.TestConfig) ... ok
test_add_message_to_queue (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_add_message_to_queue_no_data (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_data_from_invalid_string (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_data_from_valid_string (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_get_items (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_remove_message_from_empty_queue (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_remove_message_from_queue (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_send_queue (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_to_dict (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_validate_data (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_validate_data_invalid_type (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_validate_string (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_validate_string_invalid_type (test_queue.models.test_queue_message.QueueMessageTestCase) ... ok
test_add_message_to_queue (test_queue.test_local_queue.TestLocalQueue) ... ok
test_add_message_to_queue_with_no_data (test_queue.test_local_queue.TestLocalQueue) ... ok
test_empty_queue (test_queue.test_local_queue.TestLocalQueue) ... ok
test_get_items_from_queue (test_queue.test_local_queue.TestLocalQueue) ... ok
test_remove_message_from_queue (test_queue.test_local_queue.TestLocalQueue) ... ok
test_send_with_data (test_queue.test_local_queue.TestLocalQueue) ... 200
ok
test_send_without_data (test_queue.test_local_queue.TestLocalQueue) ... ok
test_add (test_queue.test_queue.TestQueue) ... ok
test_add_no_data (test_queue.test_queue.TestQueue) ... ok
test_empty (test_queue.test_queue.TestQueue) ... ok
test_get_items (test_queue.test_queue.TestQueue) ... ok
test_remove (test_queue.test_queue.TestQueue) ... ok
test_send_no_data (test_queue.test_queue.TestQueue) ... ok
test_send_with_data (test_queue.test_queue.TestQueue) ... ok
test_bad_request_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_conflict_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_forbidden_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_internal_server_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_not_found_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_service_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_timeout_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_too_many_request_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_unauthorized_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_unprocessable_error (test_resource_errors.test_errors.TestServiceErrors) ... ok
test_get_body_text (test_storage.abstract.test_file_entity.TestFileEntity) ... ok
test_get_remote_url (test_storage.abstract.test_file_entity.TestFileEntity) ... ok
test_get_stream (test_storage.abstract.test_file_entity.TestFileEntity) ... ok
test_init (test_storage.abstract.test_file_entity.TestFileEntity) ... ok
test_upload (test_storage.abstract.test_file_entity.TestFileEntity) ... ok
test_get_container (test_storage.abstract.test_storage_client.TestStorageClient) ... ok
test_get_file (test_storage.abstract.test_storage_client.TestStorageClient) ... ok
test_get_file_from_url (test_storage.abstract.test_storage_client.TestStorageClient) ... ok
test_create_file (test_storage.abstract.test_storage_container.TestStorageContainer) ... ok
test_init (test_storage.abstract.test_storage_container.TestStorageContainer) ... ok
test_list_files (test_storage.abstract.test_storage_container.TestStorageContainer) ... ok
test_azure_provider_invalid_connection_string (test_storage.config.test_storage_config.TestConfig) ... ok
test_azure_provider_missing_queue_name (test_storage.config.test_storage_config.TestConfig) ... ok
test_azure_provider_valid_connection_string (test_storage.config.test_storage_config.TestConfig) ... ok
test_non_azure_provider (test_storage.config.test_storage_config.TestConfig) ... ok
test_default_method (test_storage.models.test_config.TestCoreConfig) ... ok
test_init_with_custom_provider (test_storage.models.test_config.TestCoreConfig) ... ok
test_init_with_default_provider (test_storage.models.test_config.TestCoreConfig) ... ok
test_get_body_text (test_storage.test_azure_file_entity.TestAzureFileEntity) ... ok
test_get_remote_url (test_storage.test_azure_file_entity.TestAzureFileEntity) ... ok
test_get_stream (test_storage.test_azure_file_entity.TestAzureFileEntity) ... ok
test_upload (test_storage.test_azure_file_entity.TestAzureFileEntity) ... ok
test_get_container (test_storage.test_azure_storage_client.TestAzureStorageClient) ... ok
test_get_file (test_storage.test_azure_storage_client.TestAzureStorageClient) ... ok
test_get_file_from_url (test_storage.test_azure_storage_client.TestAzureStorageClient) ... ok
test_create_file (test_storage.test_azure_storage_container.TestAzureStorageContainer) ... ok
test_list_files (test_storage.test_azure_storage_container.TestAzureStorageContainer) ... ok
test_list_files_with_name_starts_with (test_storage.test_azure_storage_container.TestAzureStorageContainer) ... ok
test_get_body_text (test_storage.test_local_file_entity.TestLocalFileEntity) ... ok
test_get_remote_url (test_storage.test_local_file_entity.TestLocalFileEntity) ... ok
test_get_stream (test_storage.test_local_file_entity.TestLocalFileEntity) ... ok
test_upload (test_storage.test_local_file_entity.TestLocalFileEntity) ... ok
test_get_container (test_storage.test_local_storage_client.TestLocalStorageClient) ... ok
test_get_container_without_name (test_storage.test_local_storage_client.TestLocalStorageClient) ... ok
test_get_file (test_storage.test_local_storage_client.TestLocalStorageClient) ... ok
test_get_file_from_url (test_storage.test_local_storage_client.TestLocalStorageClient) ... ok
test_create_file (test_storage.test_local_storage_container.LocalStorageContainerTests) ... ok
test_list_files (test_storage.test_local_storage_container.LocalStorageContainerTests) ... ok
test_init (test_topic.abstract.test_topic_abstract.TestDerivedTopic) ... ok
test_publish (test_topic.abstract.test_topic_abstract.TestDerivedTopic) ... ok
test_subscribe (test_topic.abstract.test_topic_abstract.TestDerivedTopic) ... ok
test_azure_provider_invalid_connection_string (test_topic.config.test_topic_config.TestConfig) ... ok
test_azure_provider_logging_levels (test_topic.config.test_topic_config.TestConfig) ... ok
test_azure_provider_valid_connection_string (test_topic.config.test_topic_config.TestConfig) ... ok
test_non_azure_provider (test_topic.config.test_topic_config.TestConfig) ... ok
test_sender_creation (test_topic.config.test_topic_config.TestConfig) ... ok
test_publish (test_topic.test_local_topic.TestLocalTopic) ... ok
test_subscribe_with_subscription (test_topic.test_local_topic.TestLocalTopic) ... ok
test_publish (test_topic.test_topic.TestTopic) ... ok
test_subscribe_with_subscription (test_topic.test_topic.TestTopic) ... ok

----------------------------------------------------------------------
Ran 174 tests in 8.175s

OK

```

Coverage Report:
```shell
> coverage report
Name                                                                         Stmts   Miss  Cover
------------------------------------------------------------------------------------------------
src/python_ms_core/__init__.py                                                  93     44    53%
src/python_ms_core/core/__init__.py                                              0      0   100%
src/python_ms_core/core/auth/__init__.py                                         0      0   100%
src/python_ms_core/core/auth/abstracts/__init__.py                               0      0   100%
src/python_ms_core/core/auth/abstracts/authorizer_abstract.py                    7      0   100%
src/python_ms_core/core/auth/models/__init__.py                                  0      0   100%
src/python_ms_core/core/auth/models/permission_request.py                       43      6    86%
src/python_ms_core/core/auth/provider/__init__.py                                0      0   100%
src/python_ms_core/core/auth/provider/hosted/__init__.py                         0      0   100%
src/python_ms_core/core/auth/provider/hosted/hosted_authorizer.py               19      0   100%
src/python_ms_core/core/auth/provider/simulated/__init__.py                      0      0   100%
src/python_ms_core/core/auth/provider/simulated/simulated_authorizer.py          9      0   100%
src/python_ms_core/core/config/__init__.py                                       0      0   100%
src/python_ms_core/core/config/config.py                                        77      0   100%
src/python_ms_core/core/logger/__init__.py                                       0      0   100%
src/python_ms_core/core/logger/abstracts/__init__.py                             0      0   100%
src/python_ms_core/core/logger/abstracts/logger_abstract.py                     21      6    71%
src/python_ms_core/core/logger/local_logger.py                                  25      0   100%
src/python_ms_core/core/logger/logger.py                                        25      0   100%
src/python_ms_core/core/queue/__init__.py                                        0      0   100%
src/python_ms_core/core/queue/abstracts/__init__.py                              0      0   100%
src/python_ms_core/core/queue/abstracts/queue_abstract.py                       12      0   100%
src/python_ms_core/core/queue/config/__init__.py                                 0      0   100%
src/python_ms_core/core/queue/config/queue_config.py                             9      0   100%
src/python_ms_core/core/queue/local_queue.py                                    32      3    91%
src/python_ms_core/core/queue/models/__init__.py                                 0      0   100%
src/python_ms_core/core/queue/models/queue_message.py                           67      4    94%
src/python_ms_core/core/queue/queue.py                                          29      0   100%
src/python_ms_core/core/resource_errors/__init__.py                             90     39    57%
src/python_ms_core/core/resource_errors/errors.py                               29      3    90%
src/python_ms_core/core/storage/__init__.py                                      0      0   100%
src/python_ms_core/core/storage/abstract/__init__.py                             0      0   100%
src/python_ms_core/core/storage/abstract/file_entity.py                         21      4    81%
src/python_ms_core/core/storage/abstract/storage_client.py                       8      0   100%
src/python_ms_core/core/storage/abstract/storage_container.py                   12      2    83%
src/python_ms_core/core/storage/config/__init__.py                               0      0   100%
src/python_ms_core/core/storage/config/storage_config.py                         8      0   100%
src/python_ms_core/core/storage/models/__init__.py                               0      0   100%
src/python_ms_core/core/storage/models/config.py                                 9      9     0%
src/python_ms_core/core/storage/providers/__init__.py                            0      0   100%
src/python_ms_core/core/storage/providers/azure/__init__.py                      0      0   100%
src/python_ms_core/core/storage/providers/azure/azure_file_entity.py            22      0   100%
src/python_ms_core/core/storage/providers/azure/azure_storage_client.py         35     20    43%
src/python_ms_core/core/storage/providers/azure/azure_storage_container.py      22      0   100%
src/python_ms_core/core/storage/providers/local/__init__.py                      0      0   100%
src/python_ms_core/core/storage/providers/local/local_file_entity.py            29      0   100%
src/python_ms_core/core/storage/providers/local/local_storage_client.py         29      0   100%
src/python_ms_core/core/storage/providers/local/local_storage_container.py      28     13    54%
src/python_ms_core/core/topic/__init__.py                                        0      0   100%
src/python_ms_core/core/topic/abstract/__init__.py                               0      0   100%
src/python_ms_core/core/topic/abstract/topic_abstract.py                         8      0   100%
src/python_ms_core/core/topic/config/__init__.py                                 0      0   100%
src/python_ms_core/core/topic/config/topic_config.py                            14      0   100%
src/python_ms_core/core/topic/local_topic.py                                    45      2    96%
src/python_ms_core/core/topic/topic.py                                          61     26    57%
------------------------------------------------------------------------------------------------
TOTAL                                                                          938    181    81%
```

### CODE REVIEW

File: src/python_ms_core/__init__.py [hunk 1]
```diff
@@ -17,11 +17,18 @@
...
-            self.config = LocalConfig()
+        if config is not None:
+            if config.upper() == LOCAL_ENV:
+                self.config = LocalConfig()
+            elif config.upper() == LOCAL_ENV:
+                self.config = CoreConfig()
+                self.__check_health()
+            else:
+                self.config = UnknownConfig(provider=config)
+                logging.error(f'Failed to initialization core.get_logger for provider: {config}')
```
Comment (susrisha) [inline src/python_ms_core/__init__.py]: Please write as `Failed to initialize core.get_logger `

File: src/python_ms_core/__init__.py [hunk 2]
```diff
@@ -39,7 +46,7 @@ def get_topic(self, topic_name: str):
         elif topic_config.provider.upper() == AZURE_ENV:
             return Topic(config=topic_config, topic_name=topic_name)
         else:
-            logging.error(f'Failed to  initialization core.get_topic for provider: {topic_config.provider}')
+            logging.error(f'Failed to initialization core.get_topic for provider: {topic_config.provider}')
```
Comment (susrisha) [inline src/python_ms_core/__init__.py]: Please write as `Failed to initialize`

File: src/python_ms_core/__init__.py [hunk 3]
```diff
@@ -48,7 +55,7 @@ def get_storage_client(self):
         elif storage_config.provider.upper() == AZURE_ENV:
             return AzureStorageClient(storage_config)
         else:
-            logging.error(f'Failed to  initialization core.get_storage_client for provider: {storage_config.provider}')
+            logging.error(f'Failed to initialization core.get_storage_client for provider: {storage_config.provider}')
```
Comment (susrisha) [inline src/python_ms_core/__init__.py]: `Failed to initialize`

Comment (susrisha) [inline src/python_ms_core/__init__.py]: `Failed to initialize`

File: src/python_ms_core/__init__.py [hunk 4]
```diff
@@ -62,7 +69,7 @@ def get_authorizer(self, config: dict = None):
         elif auth_config.provider.upper() == HOSTED_ENV:
             return HostedAuthorizer(config=auth_config)
         else:
-            logging.error(f'Failed to  initialization core.get_authorizer for provider: {auth_config.provider}')
+            logging.error(f'Failed to initialization core.get_authorizer for provider: {auth_config.provider}')
```
Comment (susrisha) [inline src/python_ms_core/__init__.py]: `Failed to initialize

**Review by susrisha — APPROVED** [review]

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
I think unittest was not added to the requirements. Following readme.md is failing on my system.

(TDEI) suresh@pavan:~/code/TDEI/TDEI-python-ms-core$ pip install -r requirements.txt 
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: azure-common==1.1.28 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 1)) (1.1.28)
Requirement already satisfied: azure-core==1.26.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 2)) (1.26.1)
Requirement already satisfied: azure-servicebus==7.0.0 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 3)) (7.0.0)
Requirement already satisfied: azure-storage-blob==12.14.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 4)) (12.14.1)
Requirement already satisfied: certifi==2022.12.7 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 5)) (2022.12.7)
Requirement already satisfied: cffi==1.15.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 6)) (1.15.1)
Requirement already satisfied: charset-normalizer==2.1.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 7)) (2.1.1)
Requirement already satisfied: cryptography==39.0.0 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 8)) (39.0.0)
Requirement already satisfied: idna==3.4 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 9)) (3.4)
Requirement already satisfied: isodate==0.6.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 10)) (0.6.1)
Requirement already satisfied: msrest==0.7.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 11)) (0.7.1)
Requirement already satisfied: oauthlib==3.2.2 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 12)) (3.2.2)
Requirement already satisfied: pycparser==2.21 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 13)) (2.21)
Requirement already satisfied: python-dotenv==0.21.0 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 14)) (0.21.0)
Requirement already satisfied: requests==2.28.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 15)) (2.28.1)
Requirement already satisfied: requests-oauthlib==1.3.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 16)) (1.3.1)
Requirement already satisfied: six==1.16.0 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 17)) (1.16.0)
Requirement already satisfied: typing_extensions==4.4.0 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 18)) (4.4.0)
Requirement already satisfied: uamqp==1.6.3 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 19)) (1.6.3)
Requirement already satisfied: urllib3==1.26.14 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 20)) (1.26.14)
Requirement already satisfied: pika==1.3.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 21)) (1.3.1)
Requirement already satisfied: httpx==0.24.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 22)) (0.24.1)
Requirement already satisfied: coverage==7.2.7 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 23)) (7.2.7)
Requirement already satisfied: html_testRunner==1.2.1 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 24)) (1.2.1)
Requirement already satisfied: psutil==5.9.5 in /home/suresh/.local/lib/python3.10/site-packages (from -r requirements.txt (line 25)) (5.9.5)
Requirement already satisfied: httpcore<0.18.0,>=0.15.0 in /home/suresh/.local/lib/python3.10/site-packages (from httpx==0.24.1->-r requirements.txt (line 22)) (0.17.2)
Requirement already satisfied: sniffio in /home/suresh/.local/lib/python3.10/site-packages (from httpx==0.24.1->-r requirements.txt (line 22)) (1.3.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/lib/python3/dist-packages (from html_testRunner==1.2.1->-r requirements.txt (line 24)) (3.0.3)
Requirement already satisfied: h11<0.15,>=0.13 in /home/suresh/.local/lib/python3.10/site-packages (from httpcore<0.18.0,>=0.15.0->httpx==0.24.1->-r requirements.txt (line 22)) (0.14.0)
Requirement already satisfied: anyio<5.0,>=3.0 in /home/suresh/.local/lib/python3.10/site-packages (from httpcore<0.18.0,>=0.15.0->httpx==0.24.1->-r requirements.txt (line 22)) (3.6.2)
(TDEI) suresh@pavan:~/code/TDEI/TDEI-python-ms-core$ unittest discover -v tests/unit_tests
unittest: command not found
(TDEI) suresh@pavan:~/code/TDEI/TDEI-python-ms-core$ cd tests/
(TDEI) suresh@pavan:~/code/TDEI/TDEI-python-ms-core/tests$ unittest discover -v tests/unit_tests
unittest: command not found

**Review by sureshgaussian — APPROVED** [review]

**Pull request merged**